src/doc/rustc/src/profile-guided-optimization.md

```diff
@@ -145,3 +145,25 @@ in Clang's documentation is therefore an interesting read for anyone who wants
 to use PGO with Rust.
 
 [clang-pgo]: https://clang.llvm.org/docs/UsersManual.html#profile-guided-optimization
+
+## Community Maintained Tools
+
+As an alternative to directly using the compiler for Profile-Guided Optimization,
+you may choose to go with `cargo-pgo`, which has an intuitive command-line API
+and saves you the trouble of doing all the manual work. You can read more about
+it in their repository accessible from this link: https://github.com/Kobzol/cargo-pgo
+
+For the sake of completeness, here are the corresponding steps using `cargo-pgo`:
+
+```bash
+# Install the binary as usual: cargo install cargo-pgo
+cargo pgo build
+LLVM_PROFILE_FILE=./target/pgo-profiles/rustc-pgo_%m_%p.profraw ./target/x86_64-unknown-linux-gnu/release/myprogram
```

Comment:
I would perhaps remove the `LLVM_PROFILE_FILE` env. var. It's only needed for programs that generate a lot of processes internally (which is not *that* common), so I think that it could be avoided in this example, to make it simpler.